test/local.mk

```
@@ -131,7 +131,5 @@ test-e2e: $(kuttl_bin) $(mc_bin) local-install provider-config ## E2E tests
 .PHONY: .e2e-test-clean
 .e2e-test-clean: export KUBECONFIG = $(KIND_KUBECONFIG)
 .e2e-test-clean:
-	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete buckets --all; else echo "no kubeconfig found"; fi
-	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete iamkeys --all; else echo "no kubeconfig found"; fi
-	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete postgresql --all; else echo "no kubeconfig found"; fi
+	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete buckets --all; kubectl delete iamkeys --all; kubectl delete postgresql --all; kubectl delete mysql --all; else echo "no kubeconfig found"; fi
```

A bit more readable :)
```suggestion
	@if [ -f $(KIND_KUBECONFIG) ]; then \
  		kubectl delete buckets --all; \
  		kubectl delete iamkeys --all; \
  		kubectl delete postgresql --all; \
  		kubectl delete mysql --all; \
	else \
		echo "no kubeconfig found"; \
	fi
```

---

definitely 😄